doc/user_guide/dynamo0p3.rst

```diff
@@ -1957,12 +1957,14 @@ arguments to inter-grid kernels are as follows:
    is an integer of type ``i_def`` and has intent ``in``.
 
 2) Include the ``cell_map`` for the current cell (column). This is
-   an integer array of rank one, type ``i_def`` and intent ``in``
+   an integer array of rank two, type ``i_def`` and intent ``in``
    which provides the mapping from the coarse to the fine mesh. It
-   has extent `ncell_f_per_c`.
+   has extent ``ncell_f_per_c_x, ncell_f_per_c_y``.
```

Comment:
I would suggest enclosing this tuple in brackets, i.e. ``(ncell_f_per_c_x, ncell_f_per_c_y)``.

Reply:
Yes, that's better.